plugins/reach/lib/model/enums/VendorServiceFeature.php

```diff
@@ -5,12 +5,13 @@
  */ 
 interface VendorServiceFeature extends BaseEnum
 {
-	const CAPTIONS              = 1;
-	const TRANSLATION           = 2;
-	const ALIGNMENT             = 3;
-	const AUDIO_DESCRIPTION     = 4;
-	const CHAPTERING            = 5;
-	const INTELLIGENT_TAGGING   = 6;
-	const DUBBING               = 7;
-	const LIVE_CAPTION          = 8;
+	const CAPTIONS                      = 1;
+	const TRANSLATION                   = 2;
+	const ALIGNMENT                     = 3;
+	const AUDIO_DESCRIPTION             = 4;
+	const CHAPTERING                    = 5;
+	const INTELLIGENT_TAGGING           = 6;
+	const DUBBING                       = 7;
+	const EXTENDED_AUDIO_DESCRIPTION    = 8;
```

Comment:
you cant insert the enum before another one which already in use and recognized in a specific way